The_Limes_Have_Gone_Moldy.txt

```diff
@@ -0,0 +1,7 @@
+This is where the text gets added when we find an author for our pirate themed book.
+
+The wind was wet and wild. 
+It caused the limes to mold.
+Then all the pirates got scurvey and died.
+Everyone thought the ghosts that haunted the treasure would get the pirates.
+Nope, it's the limes.
```

Comment:
GHOST LIMES???
I'll be stealing this for a D&D campaign, if that's ok with you :)